### PR TITLE
habib | khalid / Forex and stocks ebook LP

### PR DESCRIPTION
**Pull request by Khalidxx**
Changes:

-   Added Forex ebook landing page
-   Added Stocks ebook landing page

## Type of change

-   [ ] Bug fix
-   [x] New feature
-   [ ] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

**Review by arome-binary — APPROVED** [review]
LGTM

**Review by arome-binary — CHANGES_REQUESTED** [review]

File: src/pages/landing/ebooks/forex.js [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+                mainHeaderImage={forexEbookImage}
+                imgWidth={601}
+                imgHeight={682}
+                bg="linear-gradient(to left, #661b20 39%, #cc363f);"
+                introSub={localize('Your free guide on')}
+                introMain={localize('How to trade the forex market')}
+                ebook_utm_code="forex-ebook"
+                bgMobile="linear-gradient(293deg, #ff4d47, #b51a31);"
+            />
+            <Introduction
```
[the remark this answers is not in the document]
Comment (Khalidxx) [reply src/pages/landing/ebooks/forex.js:48]: Oh yeah! Fixing it now 👍

File: src/components/custom/_agreement-label.js [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+        <label
+            style={{
+                fontWeight: 'normal',
+                lineHeight: '1px',
+                marginTop: '5px',
+                marginBottom: '20px',
+            }}
+        >
+            <Checkbox
+                style={{
```
Comment (mustofa-binary) [inline src/components/custom/_agreement-label.js:38]:
```suggestion
                you can expand styled component styled(Checkbox)`
                    ....your style here
                `
```

File: src/components/layout/ebooks/get-ebook.js [hunk 3]
```diff
@@ -0,0 +1,401 @@
...
+    border-radius: 4px;
+    display: flex;
+    justify-content: center;
+    background-color: var(--color-white);
+    border: solid 1px var(--color-grey-7);
+    min-height: 4rem;
+    height: 40px;
+    font-weight: normal;
+
+    /* margin-left: 0.8rem; */
```
Comment (mustofa-binary) [inline src/components/layout/ebooks/get-ebook.js:88]: 👀

**Review by sean-binary — APPROVED** [review]

**Pull request merged**